backend/src/tasks/ecs-client.ts

```diff
@@ -83,6 +83,7 @@ class ECSClient {
             `CENSYS_API_ID=${process.env.CENSYS_API_ID}`,
             `CENSYS_API_SECRET=${process.env.CENSYS_API_SECRET}`,
             `WORKER_USER_AGENT=${process.env.WORKER_USER_AGENT}`,
+            `SHODAN_API_KEY=${process.env.SHODAN_API_KEY}`,
```

Comment:
We need to add a SSM variable for SHODAN_API_KEY to terraform as well, and pass it in to the container definition so it's available as an environment variable from the worker.